docs/features/kubernetes/configuration.md

```diff
@@ -143,6 +150,43 @@ See also
 https://github.com/backstage/backstage/tree/master/plugins/kubernetes/src/utils/clusterLinks/formatters
 for real examples.
 
+##### `clusters.\*.dashboardParameters` (optional)
+
+Specifies additional information for the selected `dashboardApp` formatter.
+
+Note that, even though `dashboardParameters` is optional, it might be mandatory
+for some dashboards, such as GKE.
+
+###### required parameters for GKE
+
+| Name        | Description                                                              |
+| ----------- | ------------------------------------------------------------------------ |
+| projectId   | the ID of the GCP project containing your Kubernetes clusters            |
+| region      | the region of GCP containing your Kubernetes clusters                    |
+| clusterName | the name of your kubernetes cluster, within your `projectId` GCP project |
+
+Note that the GKE cluster locator can automatically provide the values for the
+`dashboardApp` and `dashboardParameters` options if you set the
+`exposeDashboard` property to `true`.
```

Comment:
Kinda curious if it's possible to remove the need for the `exposeDashboard` config? I'm thinking for example if `dashboardApp` is `gke` then the locator will populate the parameters if they're missing?

Reply:
it was made in the opposite direction: when you set exposeDashboard to true it will set dashboardApp to gke and populate dashboardParameters with projectId, region and clusterName.

Note also that when you use the GkeClusterLocator, you don't provide cluster details since they are discovered, so you can't explicitely set dashboardApp here.

Reply:
Alright, makes sense, wasn't clear on where `exposeDashboard` was supposed to be set.